app/src/main/kotlin/com/wire/android/ui/home/sync/FeatureFlagNotificationViewModel.kt

```diff
@@ -147,6 +149,16 @@ class FeatureFlagNotificationViewModel @Inject constructor(
         }
     }
 
+    private fun setE2EIRequiredState(userId: UserId) = viewModelScope.launch {
```

Comment:
```suggestion
    private fun setE2EIdRequiredState(userId: UserId) = viewModelScope.launch {
```
Missing the `d` from `Id` here as well